apps/labs/posts/announcing-labs-annual-report-2022.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Quansight Labs Annual Report 2022: Celebrating Growth and Sustainability in Open Source'
-author: pavithra-eswaramoorthy
+authors: [pavithra-eswaramoorthy]
```

Comment:
Per https://github.com/Quansight/Quansight-website/issues/364#issuecomment-1419661840, let's add all of the multiple authors on this post to use for testing.

```suggestion
authors: [pavithra-eswaramoorthy, noa-tamir, ralf-gommers, tania-allard, irina-fumarel]
```